Extension/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.18.4: November 14, 2023
+### Bug Fixes:
+* Fix 'Extract to function' not scrolling to and selecting the added header declaration. [#11676](https://github.com/microsoft/vscode-cpptools/issues/11676)
+* Fix the extension sometimes failing to activate with VS Code versions less than 1.18. [#11680](https://github.com/microsoft/vscode-cpptools/issues/11680)
```

Comment:
1.18 should be 1.85 -- I fixed it on the release page and we can update it for 1.19.